intake/interface/source/defined_plots.py

```diff
@@ -1,27 +1,29 @@
 #-----------------------------------------------------------------------------
-# Copyright (c) 2012 - 2019, Anaconda, Inc. and Intake contributors
+# Copyright (c) 2012 - 2022, Anaconda, Inc. and Intake contributors
```

Comment:
It's fine to change this, but I am told by people who know legal things much better than I that it's not necessary